Automattic/wp-calypso
# Add Monthly to Plans Grid

#### Proposed Changes

* This adds the monthly toggle for the Plans grid on the Signup flow and Plans page.

<img width="320" alt="Screenshot on 2022-06-30 at 17-32-40" src="https://user-images.githubusercontent.com/2749938/176704130-acbb0bb4-8d4f-47a4-8049-d345001855d4.png">
<img width="320" alt="Screenshot on 2022-06-30 at 17-31-36" src="https://user-images.githubusercontent.com/2749938/176704152-b44d2cdc-228a-4ef4-8c2e-1c20cd21ba99.png">

#### TODOs
- [x] recheck the monthly Pro plan once the backend work is done
- [ ] the discount is disabled by passing an empty `plans` array to the `IntervalTypeToggle`. Will probably need to pass the correct plan slugs and make sure the calculations work after the prices are fetched from the backend.
- [x] show the toggle behind an experiment check
- [ ] (Optional) use strikethrough text for the annual plan only features and mention they're available on the annual plan.
- [ ] (Optional) use a better way to show the Starter Monthly plan is unavailable.
- [ ] (Optional) use the same width for the toggle (now the toggle Plans page is wider) 

#### Testing Instructions



* Make sure your sandbox is up to date.
* Apply D83557-code to your sandboxed `public-api.wordpress.com`.
  * You may need to `rebase` the arcpatched branch to get the experiment working.
* Assign yourself to the `treatment` group of the `calypso_show_interval_type_selector_2022_07` experiment.
* Create a site and you will see the interval type selector on the plans step.
* There will be the same selector on the My Site plans page.
  <img width="876" alt="image" src="https://user-images.githubusercontent.com/212034/178274130-907a2ba7-79b0-451a-9ab0-158989d5a65e.png">
* Pick the monthly Pro plan and the purchase should be successfully processed.



Related to #
